eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/processors/SubSnpNoHgvsToDbsnpVariantsWrapperProcessorTest.java

```diff
@@ -301,6 +302,23 @@ public void transformSnpReverseContigRsAndSs() throws Exception {
         assertProcessedVariant(subSnpNoHgvs, variants.get(0), "G", "C");
     }
 
+    @Test
+    public void transformDeletionUnknonwOrientation() throws Exception {
+        String reference = "A";
+        SubSnpNoHgvs subSnpNoHgvs = new SubSnpNoHgvs(820982442L, 794525917L, reference, "T/-", ASSEMBLY, BATCH_HANDLE,
+                                                     BATCH_NAME, CHROMOSOME, CHROMOSOME_START, CONTIG_NAME,
+                                                     CONTIG_START, DbsnpVariantType.DIV, Orientation.FORWARD,
+                                                     Orientation.REVERSE, Orientation.UNKNOWN, false, false, false,
+                                                     false, CREATED_DATE, CREATED_DATE, TAXONOMY);
+
+        List<DbsnpSubmittedVariantEntity> variants = processor.process(subSnpNoHgvs).getSubmittedVariants();
+
+        assertEquals(1, variants.size());
+        assertEquals(reference, variants.get(0).getReferenceAllele());
+        assertEquals("", variants.get(0).getAlternateAllele());
+        assertFalse(variants.get(0).isAllelesMatch());
```

Comment:
We could also check that it doesn't affect other flags, or check by example, manually creating the appropriate `DbsnpSubmittedVariantEntity` to compare against.